test/jasmine/tests/geo_test.js

```diff
@@ -1421,66 +1381,6 @@ describe('Test geo interactions', function() {
         .then(done, done.fail);
     });
 
-    it('should plot to scope defaults when user setting lead to NaN map bounds', function(done) {
-        var gd = createGraphDiv();
-
-        spyOn(Lib, 'warn');
-
-        Plotly.newPlot(gd, [{
-            type: 'scattergeo',
-            lon: [0],
-            lat: [0]
-        }], {
-            geo: {
-                projection: {
-                    type: 'kavrayskiy7',
-                    rotation: {
-                        lat: 38.794799,
-                        lon: -81.622334,
-                    }
-                },
-                center: {
-                    lat: -81
-                },
-                lataxis: {
-                    range: [38.794799, 45.122292]
-                },
-                lonaxis: {
-                    range: [-82.904731, -81.622334]
-                }
-            },
-            width: 700,
-            heigth: 500
-        })
```

Comment:
This case looks like it's actually broken in this PR. If I create the above plot on `latest` I get:

![Screen Shot 2021-01-20 at 12 51 26 PM](https://user-images.githubusercontent.com/2678795/105214700-56e61d80-5b1e-11eb-89b7-8e87f93909ad.png)

But on this branch I get no warning and a completely blank plot.

Reply:
The default view is the fallback resulted from catching `d3-v3.geo` error.
We get a blank view here as we are zoomed into the graph with that particular setup.
On could zoom out by clicking on the modebar.
So it looks like we could remove the fallback and the related test by https://github.com/plotly/plotly.js/pull/5112/commits/a046871d6931e6c8f33a5441c6e6ffc385e114b9.

Reply:
OK, I guess this makes sense. The map is still in a somewhat broken state at that point: if you scroll-zoom until you can see the map appear from above, you can drag it up and down but not left/right. If you then (or instead) use the zoom out button, now you can drag left/right. Can we detect and fix this in the scroll zoom handler perhaps?

Reply:
I created an issue (see #5787) to track this edge case so that we could move on.